CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.32.2
```

Comment:
I'd vote this should be a minor as removing default sample rates is quite an important change. 

```suggestion
## 7.33.0
```

Reply:
Sounds good. I also added a note explaining the change.